### PR TITLE
Clean up incorrect/unused variable assignments

### PR DESCRIPTION
**Pull request by MichaelDeciso**
This fixes several problems regarding variable assignments or initialisation. Most of these are variables that are declared but never used or declared but immediately overwritten. It also contains cases of an assignment operator instead of a compound string concatenate operator, and some cases of (nonworking) "C-style" variable declaration.

### CODE REVIEW

File: src/etc/inc/filter.inc [hunk 1]
```diff
@@ -323,8 +323,8 @@ function filter_configure_sync($verbose = false, $flush_states = false)
     }
 
     $natrules = "\n# NAT Redirects\n";
-    $natrules = "no nat proto carp all\n";
-    $natrules = "no rdr proto carp all\n";
+    $natrules .= "no nat proto carp all\n";
```
Comment (fichtner) [inline src/etc/inc/filter.inc]: oO

Comment (MichaelDeciso) [reply src/etc/inc/filter.inc]: ... yeah, that was my reaction as well.

**Comment by fichtner** [comment]
merged. thanks! I'll add the filter thingy directly to 18.7.9

**Comment by MichaelDeciso** [comment]
Great!
Not sure if it's necessary, but the error was introduced in https://github.com/opnsense/core/commit/34e7a98df314c7e90f7e6fa87a48b613b5064c7c

**Comment by fichtner** [comment]
Yeah, I remember it vividly, but didn't do great review apparently.

**Comment by MichaelDeciso** [comment]
Oh, don't be too hard on yourself - it's very hard to spot.

**Pull request merged**